File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1099.rb
n = gets.to_i
i=0
while i<n do
    ns = gets.chomp().split(' ')
    x = ns[0].to_i
    y = ns[1].to_i
    si=0
    if(x>y) then
        temp=x
        x=y
        y=temp
    end
    if(x%2==0) then
        x=x+1
    else
        x=x+2
    end
    j=x
    while j<y do
        si=si+j
        j=j+2
    end
    puts si
    i=i+1
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1099.cs
using System;

class URI
{
    static void Main() {
        int n = int.Parse(System.Console.ReadLine().Trim()), x, y, si, temp, i, j;
        for(i=0;i<n;i++){
            string s = System.Console.ReadLine().Trim();
            string[] ns = s.Split(' ');
            x = int.Parse(ns[0]);
            y = int.Parse(ns[1]);
            si=0;
            if(x>y){
                temp=x;
                x=y;
                y=temp;
            }
    
            if(x%2==0){
                x++;
            } else{
                x+=2;
            }
            for(j=x;j<y;j+=2){
                si+=j;
            }
            Console.Write("{0}\n", si);
        }
    }
}